images/openwisp_websocket/Dockerfile

```diff
@@ -1,12 +1,13 @@
 # hadolint ignore=DL3007
-FROM openwisp/openwisp-base:latest
+ARG BASE_IMAGE=openwisp/openwisp-base:latest
+FROM ${BASE_IMAGE}
 
 WORKDIR /opt/openwisp/
 
-COPY --chown=openwisp:root  ./common/ \
```

Comment:
Why are we removing this?
If I remember correctly, the intention of making `openwisp` the owner was because we did not want to run things as root in production.

Reply:
Hello, thank you for the feedback. Internal user is great until you try to bind volumes from host's system (our case). OpenWISP user has fixed UID 1001 that might mean something on the host and it will mess things up. It is much preferable to specify container's runtime user either via `UID` and `GID` variables or via `user` option in docker(compose). The first approach is used in all `bitnami` images but I am a big fan of the later. But both works in this case.